OpenRA.Mods.Common/Traits/Explodes.cs

```diff
@@ -81,5 +90,14 @@ WeaponInfo ChooseWeaponForExplosion(Actor self)
 			var useFullExplosion = self.World.SharedRandom.Next(100) <= info.LoadedChance;
 			return (shouldExplode && useFullExplosion) ? info.WeaponInfo : info.EmptyWeaponInfo;
 		}
+
+		public void Damaged(Actor self, AttackInfo e)
+		{
+			if (info.DamageThreshold == 0)
+				return;
+
+			if ((health.HP * 100) < (100 - info.DamageThreshold) * health.MaxHP)
```

Comment:
This makes the damage threshold a count down (e.g. "explode when i'm damaged more than 10%"), while our convention and the description in the TraitInfo is to specify the *remaining* health.  Can you change this to `info.DamageThreshold * health.MaxHP`, and then adjust the rules to 90?